### DISCUSSION TITLE
Failed to find class App\GraphQl\Queries\TodoQuery in namespaces [] for directive @paginate

### DISCUSSION THREAD

**Discussion by mwangaben** [nuwave/lighthouse#2548] [Q&A]
I get an error when run `php artisan lighthouse:validate-schema` on production serve but this does not happen on local development 

on local development i am using Macos Ventura and php8.2.16 
on production serve  iam using Debian 10  and php8.2.18 

i have tested even on the another  machine that runs  Macos Ventura too but no error.

Please could it be a specific  php extention that my be missing on my production serve or i am missing something here

<img width="783" alt="Screenshot 2024-04-25 at 05 39 12" src="https://github.com/nuwave/lighthouse/assets/17113999/2efc2cbd-9d78-4299-adad-35d489503c60">

on development all tests are passing as per below image

<img width="469" alt="Screenshot 2024-04-25 at 05 45 44" src="https://github.com/nuwave/lighthouse/assets/17113999/32de9e6b-97b1-4c25-8c87-b136030e55f7">

**Comment by spawnia** [comment]
From the information you gave I can not tell this for certain, but suspect this to be an issue with case sensitivity. Make sure to capitalize `GraphQL` consistently, the error you gave shows `GraphQl`.
